5_vision_language_models/vlm_usage.md

```diff
@@ -0,0 +1 @@
+# Visual Language Models
```

Comment:
This can be similar to the 'Use SmolVLM with Transformers' section from this [blog post](https://huggingface.co/blog/smolvlm) basically just showing inference.